dpgen/generator/arginfo.py

```diff
@@ -160,6 +188,8 @@ def model_devi_lmp_args() -> List[Argument]:
     doc_model_devi_clean_traj = 'If type of model_devi_clean_traj is bool type then it denote whether to clean traj folders in MD since they are too large. If it is Int type, then the most recent n iterations of traj folders will be retained, others will be removed.'
     doc_model_devi_merge_traj = 'If model_devi_merge_traj is set as True, only all.lammpstrj will be generated, instead of lots of small traj files.'
     doc_model_devi_nopbc = 'Assume open boundary condition in MD simulations.'
+    doc_model_devi_plumed = '' # looking forward to update
```

Comment:
Actually, I'm not sure if this parameter is functional at all. Because to run model_devi with plumed, we use "template", which is a completely independent setting and is not at all affected by this parameter.